spec/03-custom-serve_spec.lua

```diff
@@ -40,22 +40,22 @@ describe("Plugin: prometheus (custom server)",function()
     end)
 
     it("metrics can be read from a different port", function()
-    	local res = assert(proxy_client:send {
-    	  method  = "GET",
-    	  path    = "/status/200",
-    	  headers = {
-    	    host = helpers.mock_upstream_host,
-    	  }
-    	})
-    	assert.res_status(200, res)
-
-			local client = helpers.http_client("127.0.0.1", 9542)
-    	local res = assert(client:send {
-    	  method  = "GET",
-    	  path    = "/metrics",
-    	})
-    	local body = assert.res_status(200, res)
-    	assert.matches('kong_http_status{code="200",service="mock-service"} 1', body, nil, true)
+      local res = assert(proxy_client:send {
```

Comment:
Can you also update tests to remove redefines of `res` and`body`?

Reply:
I had a quick go at it but it made things uglier, I opted to leave it for someone with more time